[2025-04-22T09:19:13.680328+00:00] http_client.INFO: Request: "GET https://data.jsdelivr.com/v1/packages/npm/@hotwired/stimulus/resolved?specifier=%5E3.0.0" [] []
[2025-04-22T09:19:13.919748+00:00] http_client.INFO: Response: "200 https://data.jsdelivr.com/v1/packages/npm/@hotwired/stimulus/resolved?specifier=%5E3.0.0" 0.22356 seconds {"http_code":200,"url":"https://data.jsdelivr.com/v1/packages/npm/@hotwired/stimulus/resolved?specifier=%5E3.0.0","total_time":0.22356} []
[2025-04-22T09:19:13.921538+00:00] http_client.INFO: Request: "GET https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm" [] []
[2025-04-22T09:19:13.921967+00:00] http_client.INFO: Request: "GET https://data.jsdelivr.com/v1/packages/npm/@hotwired/stimulus@3.2.2/entrypoints" [] []
[2025-04-22T09:19:14.120495+00:00] http_client.INFO: Response: "200 https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm" 0.190233 seconds {"http_code":200,"url":"https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm","total_time":0.190233} []
[2025-04-22T09:19:14.122723+00:00] http_client.INFO: Response: "200 https://data.jsdelivr.com/v1/packages/npm/@hotwired/stimulus@3.2.2/entrypoints" 0.033847 seconds {"http_code":200,"url":"https://data.jsdelivr.com/v1/packages/npm/@hotwired/stimulus@3.2.2/entrypoints","total_time":0.033847} []
[2025-04-22T09:19:14.125692+00:00] http_client.INFO: Request: "GET https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm" [] []
[2025-04-22T09:19:14.157079+00:00] http_client.INFO: Response: "200 https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm" 0.029658 seconds {"http_code":200,"url":"https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm","total_time":0.029658} []
[2025-04-22T09:19:15.352416+00:00] http_client.INFO: Request: "GET https://data.jsdelivr.com/v1/packages/npm/@hotwired/turbo/resolved?specifier=%5E7.1.0" [] []
[2025-04-22T09:19:15.509568+00:00] http_client.INFO: Response: "200 https://data.jsdelivr.com/v1/packages/npm/@hotwired/turbo/resolved?specifier=%5E7.1.0" 0.148022 seconds {"http_code":200,"url":"https://data.jsdelivr.com/v1/packages/npm/@hotwired/turbo/resolved?specifier=%5E7.1.0","total_time":0.148022} []
[2025-04-22T09:19:15.510207+00:00] http_client.INFO: Request: "GET https://cdn.jsdelivr.net/npm/@hotwired/turbo@7.3.0/+esm" [] []
[2025-04-22T09:19:15.510449+00:00] http_client.INFO: Request: "GET https://data.jsdelivr.com/v1/packages/npm/@hotwired/turbo@7.3.0/entrypoints" [] []
[2025-04-22T09:19:15.645585+00:00] http_client.INFO: Response: "200 https://cdn.jsdelivr.net/npm/@hotwired/turbo@7.3.0/+esm" 0.129385 seconds {"http_code":200,"url":"https://cdn.jsdelivr.net/npm/@hotwired/turbo@7.3.0/+esm","total_time":0.129385} []
[2025-04-22T09:19:15.652449+00:00] http_client.INFO: Response: "200 https://data.jsdelivr.com/v1/packages/npm/@hotwired/turbo@7.3.0/entrypoints" 0.030245 seconds {"http_code":200,"url":"https://data.jsdelivr.com/v1/packages/npm/@hotwired/turbo@7.3.0/entrypoints","total_time":0.030245} []
[2025-04-22T09:19:15.654465+00:00] http_client.INFO: Request: "GET https://cdn.jsdelivr.net/npm/@hotwired/turbo@7.3.0/+esm" [] []
[2025-04-22T09:19:15.686935+00:00] http_client.INFO: Response: "200 https://cdn.jsdelivr.net/npm/@hotwired/turbo@7.3.0/+esm" 0.030886 seconds {"http_code":200,"url":"https://cdn.jsdelivr.net/npm/@hotwired/turbo@7.3.0/+esm","total_time":0.030886} []
[2025-04-22T09:31:47.664870+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-22T09:31:47.748812+00:00] request.INFO: Matched route "_wdt_stylesheet". {"route":"_wdt_stylesheet","route_parameters":{"_route":"_wdt_stylesheet","_controller":"web_profiler.controller.profiler::toolbarStylesheetAction"},"request_uri":"http://localhost:8080/_wdt/styles","method":"GET"} []
[2025-04-22T09:31:47.784880+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b61fbd"},"request_uri":"http://localhost:8080/_wdt/b61fbd?XDEBUG_IGNORE=1","method":"GET"} []
[2025-04-22T09:33:05.908223+00:00] request.INFO: Matched route "app_index_index". {"route":"app_index_index","route_parameters":{"_route":"app_index_index","_controller":"App\\Controller\\IndexController::index"},"request_uri":"http://localhost:8080/index","method":"GET"} []
[2025-04-22T09:33:05.910429+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2025-04-22T09:33:05.984127+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/favicon.ico" (from "http://localhost:8080/index")" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/favicon.ico\" (from \"http://localhost:8080/index\") at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2025-04-22T12:30:17.782516+00:00] php.DEBUG: Warning: filemtime(): stat failed for /var/www/symfony_docker/config/routes/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/symfony/config/Resource/FileResource.php","line":58,"trace":[{"file":"/home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":44,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2025-04-23T10:25:09.450769+00:00] doctrine.INFO: Connecting with parameters array{"use_savepoints":true,"driver":"pdo_mysql","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"use_savepoints":true,"driver":"pdo_mysql","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:25:09.457994+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:25:09.458114+00:00] doctrine.INFO: Connecting with parameters array{"use_savepoints":true,"driver":"pdo_mysql","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"use_savepoints":true,"driver":"pdo_mysql","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:25:09.459652+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:25:09.460192+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:29:06.817169+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:29:06.822487+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:29:06.822586+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:29:06.823540+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:29:06.824012+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:32:24.355416+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:32:24.357402+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:32:24.357475+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:32:24.358220+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:32:24.358517+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:34:11.957150+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:34:11.959961+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:34:11.960058+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:34:11.961052+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:34:11.961317+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:35:06.550419+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:35:06.555898+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:35:06.555991+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:35:06.557830+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:35:06.558270+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:35:55.160433+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:35:55.162809+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:35:55.162908+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:35:55.163770+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:35:55.164016+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:38:12.476978+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"dbname":"symfony_docker","charset":"utf8mb4"}} []
[2025-04-23T10:38:12.479139+00:00] deprecation.INFO: User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Relying on a fallback connection used to determine the database platform while connecting to a non-existing database is deprecated. Either use an existing database name in connection parameters or omit the database name if the platform and the server configuration allow that. (Connection.php:464 called by Connection.php:416, https://github.com/doctrine/dbal/pull/5707, package doctrine/dbal) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:38:12.479262+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"} {"params":{"driver":"pdo_mysql","use_savepoints":true,"idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":{"collation":"utf8mb4_unicode_ci"},"charset":"utf8mb4"}} []
[2025-04-23T10:38:12.480515+00:00] console.CRITICAL: Error thrown while running command "doctrine:database:create". Message: "An exception occurred in the driver: could not find driver" {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:118)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:24)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/dbal/src/Driver/PDO/MySQL/Driver.php:33)","command":"doctrine:database:create","message":"An exception occurred in the driver: could not find driver"} []
[2025-04-23T10:38:12.480775+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T10:56:20.761988+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T10:56:20.774653+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"localhost","port":4306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T10:56:20.779728+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T11:51:01.862516+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T11:51:01.873356+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"localhost","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"localhost","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T11:51:01.875934+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T11:51:25.824803+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T11:51:25.832886+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T11:51:25.834661+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T11:54:53.833212+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T11:54:53.842317+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T11:54:53.844395+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T11:58:34.173729+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T11:58:34.182488+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T11:58:34.184314+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T12:01:27.182189+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T12:01:27.190711+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T12:01:27.192599+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T12:07:50.549546+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T12:07:50.562542+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T12:07:50.567443+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T12:14:54.033194+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /var/www/symfony_docker/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T12:14:54.042115+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T12:14:54.049502+00:00] doctrine.DEBUG: Executing statement: CREATE DATABASE `symfony_docker` {"sql":"CREATE DATABASE `symfony_docker`"} []
[2025-04-23T12:14:54.052691+00:00] doctrine.INFO: Disconnecting [] []
[2025-04-23T12:14:54.053557+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T12:15:02.663362+00:00] deprecation.INFO: User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. "8.0.31" instead of "8". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) {"exception":"[object] (ErrorException(code: 0): User Deprecated: Version detection logic for MySQL will change in DBAL 4. Please specify the version as the server reports it, e.g. \"8.0.31\" instead of \"8\". (AbstractMySQLDriver.php:89 called by AbstractDriverMiddleware.php:68, https://github.com/doctrine/dbal/pull/5779, package doctrine/orm) at /home/mateusz/Symfony/Radio-Amateur-Toolki/app/vendor/doctrine/deprecations/src/Deprecation.php:208)"} []
[2025-04-23T12:15:02.671522+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"}}} []
[2025-04-23T12:15:02.673137+00:00] console.DEBUG: Command "doctrine:database:create" exited with code "1" {"command":"doctrine:database:create","code":1} []
[2025-04-23T12:19:23.492923+00:00] deprecation.INFO: User Deprecated: Since doctrine/doctrine-bundle 2.2: The "Doctrine\Bundle\DoctrineBundle\Command\Proxy\RunSqlDoctrineCommand" (doctrine:query:sql) is deprecated, use dbal:run-sql command instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since doctrine/doctrine-bundle 2.2: The \"Doctrine\\Bundle\\DoctrineBundle\\Command\\Proxy\\RunSqlDoctrineCommand\" (doctrine:query:sql) is deprecated, use dbal:run-sql command instead. at /var/www/symfony_docker/vendor/doctrine/doctrine-bundle/src/Command/Proxy/RunSqlDoctrineCommand.php:34)"} []
[2025-04-23T12:19:23.496091+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"},"dbname":"symfony_docker"} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"},"dbname":"symfony_docker"}} []
[2025-04-23T12:19:23.501403+00:00] doctrine.DEBUG: Executing query: SELECT 1 {"sql":"SELECT 1"} []
[2025-04-23T12:19:23.517558+00:00] doctrine.INFO: Disconnecting [] []
[2025-04-23T12:19:35.571655+00:00] console.CRITICAL: Error thrown while running command "'doctrine:database:info'". Message: "Command "doctrine:database:info" is not defined.  Did you mean one of these?     doctrine:database:create     doctrine:database:drop     doctrine:mapping:info" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"doctrine:database:info\" is not defined.\n\nDid you mean one of these?\n    doctrine:database:create\n    doctrine:database:drop\n    doctrine:mapping:info at /var/www/symfony_docker/vendor/symfony/console/Application.php:726)","command":"'doctrine:database:info'","message":"Command \"doctrine:database:info\" is not defined.\n\nDid you mean one of these?\n    doctrine:database:create\n    doctrine:database:drop\n    doctrine:mapping:info"} []
[2025-04-25T08:48:10.729262+00:00] deprecation.INFO: User Deprecated: Since doctrine/doctrine-bundle 2.2: The "Doctrine\Bundle\DoctrineBundle\Command\Proxy\RunSqlDoctrineCommand" (doctrine:query:sql) is deprecated, use dbal:run-sql command instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since doctrine/doctrine-bundle 2.2: The \"Doctrine\\Bundle\\DoctrineBundle\\Command\\Proxy\\RunSqlDoctrineCommand\" (doctrine:query:sql) is deprecated, use dbal:run-sql command instead. at /var/www/symfony_docker/vendor/doctrine/doctrine-bundle/src/Command/Proxy/RunSqlDoctrineCommand.php:34)"} []
[2025-04-25T08:48:10.736293+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"},"dbname":"symfony_docker"} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"},"dbname":"symfony_docker"}} []
[2025-04-25T08:48:10.742480+00:00] doctrine.DEBUG: Executing query: SELECT 1 {"sql":"SELECT 1"} []
[2025-04-25T08:48:10.752307+00:00] doctrine.INFO: Disconnecting [] []
[2025-04-25T10:00:33.379629+00:00] deprecation.INFO: User Deprecated: Since doctrine/doctrine-bundle 2.2: The "Doctrine\Bundle\DoctrineBundle\Command\Proxy\RunSqlDoctrineCommand" (doctrine:query:sql) is deprecated, use dbal:run-sql command instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Since doctrine/doctrine-bundle 2.2: The \"Doctrine\\Bundle\\DoctrineBundle\\Command\\Proxy\\RunSqlDoctrineCommand\" (doctrine:query:sql) is deprecated, use dbal:run-sql command instead. at /var/www/symfony_docker/vendor/doctrine/doctrine-bundle/src/Command/Proxy/RunSqlDoctrineCommand.php:34)"} []
[2025-04-25T10:00:33.391085+00:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"},"dbname":"symfony_docker"} {"params":{"driver":"pdo_mysql","charset":"utf8mb4","idle_connection_ttl":600,"host":"database","port":3306,"user":"symfony","password":"<redacted>","driverOptions":[],"serverVersion":"8.0","defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci"},"dbname":"symfony_docker"}} []
[2025-04-25T10:00:33.409157+00:00] doctrine.DEBUG: Executing query: SELECT 1 {"sql":"SELECT 1"} []
[2025-04-25T10:00:33.440024+00:00] doctrine.INFO: Disconnecting [] []
[2025-04-25T10:08:32.001299+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpTTObqJ" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpTTObqJ,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpTTObqJ\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpTTObqJ,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:08:32.142233+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/c0/66)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/c0/66). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:34.892014+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.php2PDaNf" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.php2PDaNf,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.php2PDaNf\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.php2PDaNf,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:34.953307+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/14/c6)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/14/c6). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:35.910927+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpTH8vxT" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpTH8vxT,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpTH8vxT\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpTH8vxT,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:35.929406+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/23/d0)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/23/d0). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:36.505685+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpNUyh9e" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpNUyh9e,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpNUyh9e\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpNUyh9e,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:36.521952+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/78/94)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/78/94). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:37.016868+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpAYCEGf" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpAYCEGf,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpAYCEGf\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpAYCEGf,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:37.033556+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/34/80)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/34/80). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:37.524746+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpS4Y6Q1" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpS4Y6Q1,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpS4Y6Q1\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpS4Y6Q1,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:37.541349+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/e4/2f)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/e4/2f). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:38.132554+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpLPHDRF" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpLPHDRF,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpLPHDRF\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpLPHDRF,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:38.149837+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/65/23)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/65/23). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:10:38.785914+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpq0b9vj" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpq0b9vj,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpq0b9vj\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpq0b9vj,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:10:38.802680+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/5d/72)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/5d/72). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:12:26.388329+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phppBNx9e" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phppBNx9e,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phppBNx9e\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phppBNx9e,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:12:26.500189+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/41/67)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/41/67). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:12:27.501856+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpCeq1lB" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpCeq1lB,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpCeq1lB\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpCeq1lB,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:12:27.522293+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/3a/68)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/3a/68). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:13:17.529763+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpA21Fhr" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpA21Fhr,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpA21Fhr\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpA21Fhr,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:13:17.640692+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/44/d2)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/44/d2). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:13:29.038457+00:00] console.CRITICAL: Error thrown while running command "cache:clear". Message: "Unable to write in the "/var/www/symfony_docker/var/cache/dev" directory." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\RuntimeException(code: 0): Unable to write in the \"/var/www/symfony_docker/var/cache/dev\" directory. at /var/www/symfony_docker/vendor/symfony/framework-bundle/Command/CacheClearCommand.php:83)","command":"cache:clear","message":"Unable to write in the \"/var/www/symfony_docker/var/cache/dev\" directory."} []
[2025-04-25T10:13:29.039787+00:00] console.DEBUG: Command "cache:clear" exited with code "1" {"command":"cache:clear","code":1} []
[2025-04-25T10:14:12.429735+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Filesystem\Exception\IOException: "Cannot rename "/tmp/url_matching_routes.phpepGYsu" to "/var/www/symfony_docker/var/cache/dev/url_matching_routes.php": rename(/tmp/url_matching_routes.phpepGYsu,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied" at Filesystem.php line 304 {"exception":"[object] (Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Cannot rename \"/tmp/url_matching_routes.phpepGYsu\" to \"/var/www/symfony_docker/var/cache/dev/url_matching_routes.php\": rename(/tmp/url_matching_routes.phpepGYsu,/var/www/symfony_docker/var/cache/dev/url_matching_routes.php): Permission denied at /var/www/symfony_docker/vendor/symfony/filesystem/Filesystem.php:304)"} []
[2025-04-25T10:14:12.496315+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/ac/3c)." at FileProfilerStorage.php line 140 {"exception":"[object] (RuntimeException(code: 0): Unable to create the storage directory (/var/www/symfony_docker/var/cache/dev/profiler/ac/3c). at /var/www/symfony_docker/vendor/symfony/http-kernel/Profiler/FileProfilerStorage.php:140)"} []
[2025-04-25T10:15:56.869136+00:00] console.CRITICAL: Error thrown while running command "cache:warmup". Message: "Invalid "xlf:Zone" locale." {"exception":"[object] (Symfony\\Component\\Translation\\Exception\\InvalidArgumentException(code: 0): Invalid \"xlf:Zone\" locale. at /var/www/symfony_docker/vendor/symfony/translation/Translator.php:414)","command":"cache:warmup","message":"Invalid \"xlf:Zone\" locale."} []
[2025-04-25T10:15:56.871155+00:00] console.DEBUG: Command "cache:warmup" exited with code "1" {"command":"cache:warmup","code":1} []
[2025-04-25T10:16:07.019734+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T10:16:07.155625+00:00] request.INFO: Matched route "_wdt_stylesheet". {"route":"_wdt_stylesheet","route_parameters":{"_route":"_wdt_stylesheet","_controller":"web_profiler.controller.profiler::toolbarStylesheetAction"},"request_uri":"http://localhost:8080/_wdt/styles","method":"GET"} []
[2025-04-25T10:16:07.200669+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"49708b"},"request_uri":"http://localhost:8080/_wdt/49708b?XDEBUG_IGNORE=1","method":"GET"} []
[2025-04-25T10:16:25.814863+00:00] request.INFO: Matched route "app_index_index". {"route":"app_index_index","route_parameters":{"_route":"app_index_index","_controller":"App\\Controller\\IndexController::index"},"request_uri":"http://localhost:8080/index","method":"GET"} []
[2025-04-25T10:16:25.826123+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2025-04-25T12:55:50.877668+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T12:56:00.359050+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T12:56:00.397761+00:00] request.INFO: Matched route "_wdt_stylesheet". {"route":"_wdt_stylesheet","route_parameters":{"_route":"_wdt_stylesheet","_controller":"web_profiler.controller.profiler::toolbarStylesheetAction"},"request_uri":"http://localhost:8080/_wdt/styles","method":"GET"} []
[2025-04-25T12:56:00.409789+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"763e5e"},"request_uri":"http://localhost:8080/_wdt/763e5e?XDEBUG_IGNORE=1","method":"GET"} []
[2025-04-25T12:56:05.430847+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T12:56:09.522994+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T12:56:09.567957+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9491c5"},"request_uri":"http://localhost:8080/_wdt/9491c5?XDEBUG_IGNORE=1","method":"GET"} []
[2025-04-25T12:56:30.643543+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T12:56:35.851292+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T12:56:45.125763+00:00] request.INFO: Matched route "app_index_index". {"route":"app_index_index","route_parameters":{"_route":"app_index_index","_controller":"App\\Controller\\IndexController::index"},"request_uri":"http://localhost:8080/index","method":"GET"} []
[2025-04-25T12:56:45.128047+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2025-04-25T13:04:09.584414+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T13:05:07.323418+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T13:05:46.540941+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T13:05:54.839365+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T13:06:18.121255+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T13:06:18.146034+00:00] request.INFO: Matched route "_wdt_stylesheet". {"route":"_wdt_stylesheet","route_parameters":{"_route":"_wdt_stylesheet","_controller":"web_profiler.controller.profiler::toolbarStylesheetAction"},"request_uri":"http://localhost:8080/_wdt/styles","method":"GET"} []
[2025-04-25T13:06:18.161179+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"6b1616"},"request_uri":"http://localhost:8080/_wdt/6b1616?XDEBUG_IGNORE=1","method":"GET"} []
[2025-04-25T13:06:19.420578+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8080/"" at RouterListener.php line 149 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8080/\" at /var/www/symfony_docker/vendor/symfony/http-kernel/EventListener/RouterListener.php:149)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /var/www/symfony_docker/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:181)"} []
[2025-04-25T13:06:19.450100+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"fbe231"},"request_uri":"http://localhost:8080/_wdt/fbe231?XDEBUG_IGNORE=1","method":"GET"} []
